config/config.js

```diff
@@ -64,6 +64,12 @@ module.exports = function (fs, path, url, convict) {
       format: 'duration',
       env: 'PRUNE_TOKENS_MAX_AGE',
     },
+    signinCodesMaxAge: {
+      doc: 'Maximum age for signinCodes, after which they will expire',
+      default: '2 days',
```

Comment:
> ...this seemed a little high as a default to me.

Fwiw, the success metric for this feature is "users who have successfully connected a mobile device within 48 hours of signing up". I just picked the shortest possible expiry time which would enable shifting that metrics. But we could also change the metrics of course...